README.md

```diff
@@ -3,20 +3,98 @@
+
+
+## Status
+
+Keystone started as an academic project that helps researchers to build and test their ideas. 
+Now, Keystone is an **Incubation Stage** open-source project of Confidential Computing Consortium (CCC) under Linux Foundation. 
+
+Keystone helped many researchers focus on their creative ideas instead of building TEE by themselves from scratch.
+This resulted in many innovative research projects and publications, which have been pushing the technical advancement of TEEs.
+
+We are currently trying to make Keystone production-ready. You can find the latest general roadmap of Keystone [here](https://docs.google.com/document/d/1E-982564GvOcWzdCqM7TXCJV_7uWy2F8NiwglWorjFA/edit#heading=h.xa3pe84ubay4)
+
+Here are some ongoing and/or planned efforts towards the goal:
+
+* **Technical Improvements**: Make Keystone more usable and on par with existing industry solutions, including memory isolation improvement, better application and hardware support, and additional features.
+
+* **Parity with Industry Standards**: Make Keystone follow the industry standard. This includes standard cryptography, measured boot, and remote attestation protocols. 
+
+* **Hardware Integration**: Partner with RISC-V hardware designer/vendor to fully integrate with the hardware. This includes integration with hardware root-of-trust, memory encryption engine, and crypto accelerators.
+
+## Documentation
 
 See [docs](http://docs.keystone-enclave.org) for getting started.
 
-# Contributing
+## Hardware Support
+
+Keystone requires standard RISC-V platform with *hardware root of trust*, which includes a secure key storage and measured boot. Currently, no hardware root of trust has been designed or manufactured specifically for Keystone. We hope to integrate Keystone with a standard root-of-trust when it becomes available.
+
+As this project focuses more on the software stack and the toolchain, you can still run the full Keystone software stack on top of a few RISC-V platforms. See https://github.com/keystone-enclave/keystone/tree/master/sm/plat for the supported platforms. In general, `generic` should work with most of the standard RISC-V cores as long as they support:
+
+- RV64 with SV39 addressing mode (or RV32 with SV32)
+- M/S/U privilege modes
+- More than 4 PMP registers
+
+For full security, platform architect needs to provide the followings
+
+- Entropy source (random number generator)
+- Measured boot
+- Secure on-chip key storage
+
+Keystone doesn’t provide high-performance hardware-based memory encryption, as it requires a proprietary memory controller. Instead, it provides software-based encryption, which uses scratchpad SRAM (if any) to encrypt physical pages.
+
+## Team
+
+Contributors and Maintainers
+
+- Gregor Haas
+- Evgeny Pobachienko
+- Jakob Sorensen
+- Dayeol Lee
+- David Kholbrenner
+- Alex Thomas
+
+Past Contributors
```

Comment:
personally I'd just lump everyone into "contributors" and _maybe_ have "Maintainers" or "Leads" as the 2-3 most active people, rather than retire folks to "past contributors". Either is fine.